opm/autodiff/SimulatorFullyImplicitBlackoilOutputEbos.hpp

```diff
@@ -743,5 +745,20 @@ namespace Opm
 
         writeTimeStepWithCellProperties(timer, localState, localWellState, cellData, substep);
     }
+
+
+    bool BlackoilOutputWriterEbos::hasFIPSummaryKeywords() const
+    {
+      const SummaryConfig& summaryConfig = eclipseState_.getSummaryConfig();
+      return detail::hasFRBKeyword(summaryConfig, "WIP")  ||
```

Comment:
ack. works now for flow_mpi, flow_ebos is not happy but that's a different issue.